railib/api.py

```diff
@@ -324,11 +324,13 @@ def _parse_arrow_results(files: List[TransactionAsyncFile]):
 def poll_with_specified_overhead(
     f,
     overhead_rate: float,
-    start_time: int = time.time(),
+    start_time: float = None,
```

Comment:
I dunno if the type checker is configured differently in the SDK but in pyrel's repo + my editor it's an error not to explicitly union with None when assigning a default of None.

Reply:
Oh yes, this is a type error. But it's all over the codebase and I figure it makes sense to be consistent and address that concern in a separate PR.